inmanta.spec

```diff
@@ -0,0 +1,83 @@
+# Use release 0 for prerelease version.
+%define release 0
+%define version 2020.6
+%define buildid .dev202012211003
+%define buildid_egg .dev202012211003
+%define venv inmanta-venv
+%define _p3 %{venv}/bin/python3
+%define _unique_build_ids 0
+%define _debugsource_packages 0
+%define _debuginfo_subpackages 0
+%define _enable_debug_packages 0
+%define debug_package %{nil}
+
+%define sourceversion %{version}%{?buildid}
+%define sourceversion_egg %{version}%{?buildid_egg}
+
+Name:           python3-inmanta
+Version:        %{version}
+
+Release:        %{release}%{?buildid}%{?tag}%{?dist}
+Summary:        Inmanta Service Orchestrator
+
+Group:          Development/Languages
+License:        EULA
+URL:            http://inmanta.com
+Source0:        inmanta-%{sourceversion_egg}.tar.gz
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires:  python3-inmanta-core
+BuildRequires:  systemd
+
+Requires:  python3-inmanta-core
```

Comment:
Shouldn't we also add `python3-inmanta-agent` and `python3-inmanta-server`?

Reply:
no, because then you cannot install only core